src/app/tests/suites/certification/Test_TC_DISHM_1_1.yaml

```diff
@@ -13,90 +13,128 @@
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "TH reads the ClusterRevision from DUT"
+      command: "readAttribute"
+      attribute: "ClusterRevision"
+      response:
+          value: 1
+          constraints:
+              type: int16u
+
+    - label: "TH reads the FeatureMap from DUT"
+      PICS: " !DISHM.S.F00 "
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          value: 0
+          constraints:
+              type: bitmap32
 
-          [1676288620.483812][4014:4016] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0050 Attribute 0x0000_FFFB DataVersion: 2649295313
-          [1676288620.483850][4014:4016] CHIP:TOO: AttributeList: 13 entries
-          [1676288620.483861][4014:4016] CHIP:TOO: [1]: 0
-          [1676288620.483870][4014:4016] CHIP:TOO: [2]: 1
-          [1676288620.483878][4014:4016] CHIP:TOO: [3]: 65528
-          [1676288620.483886][4014:4016] CHIP:TOO: [4]: 65529
-          [1676288620.483892][4014:4016] CHIP:TOO: [5]: 65530
-          [1676288620.483899][4014:4016] CHIP:TOO: [6]: 65531
-          [1676288620.483906][4014:4016] CHIP:TOO: [7]: 4293984257
-          [1676288620.483915][4014:4016] CHIP:TOO: [8]: 2
-          [1676288620.483922][4014:4016] CHIP:TOO: [9]: 3
-          [1676288620.483929][4014:4016] CHIP:TOO: [10]: 4
-          [1676288620.483934][4014:4016] CHIP:TOO: [11]: 5
-          [1676288620.483941][4014:4016] CHIP:TOO: [12]: 65532
-          [1676288620.483947][4014:4016] CHIP:TOO: [13]: 65533
-          [1676288620.484034][4014:4016] CHIP:EM: <<< [E:29816i M:167554497 (Ack:41020138)] (S) Msg TX to 1:000000000
-
-
-          *NOTE* : This sample attribute list reflects the currently implemented attributes on the all-clusters-app.
-      disabled: true
-
-    - label: "Step 5: TH reads from the DUT the AcceptedCommandList attribute."
-      verification: |
-          ./chip-tool dishwashermodeselect read accepted-command-list 1 1
-
-          Verify the "AcceptedCommandList" contains feature dependent commands
-          if !MOD.S.F02(DEPCTM) feature supports list includes(value 0x0000) and
-          if MOD.S.F01(EXT) feature supports list includes(value 0x0001) on the TH (chip-tool) and below is the sample log provided for the raspi platform:
+    - label:
+          "Given DISHM.S.F00(DEPONOFF) ensure featuremap has the correct bit set"
+      PICS: DISHM.S.F00
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          value: 1
+          constraints:
+              type: bitmap32
+    # Reference for "SHALL NOT contain" issue: https://github.com/project-chip/connectedhomeip/issues/28287
+    - label: "4. TH reads AttributeList from DUT"
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [0, 1, 65528, 65529, 65530, 65531, 65532, 65533]
```

Comment:
Your CI is failing here: 

`***** Test Failure: value expect the value 65530 but the list does not contains it.`

I _think_ this is FFFA event list, so either that shouldn't be there or its not switched on for the cluster, its in test plan so maybe the latter!

Reply:
Yeah I believe the latter is correct as well. I'm not sure what to do. Suppose I should make an issue for the sdk?

Reply:
Reference Issue: https://github.com/project-chip/connectedhomeip/issues/28525

Reply:
event list was unprovisional, then it went provisional again. You can just remove it from the list here, it's fine.